src/polar/Pie.tsx

```diff
@@ -163,6 +163,9 @@ type PiePayloadInputProps = {
 };
 
 const computeLegendPayloadFromPieData = ({ sectors, legendType }: PiePayloadInputProps): Array<LegendPayload> => {
+  if (sectors == null) {
```

Comment:
This is because Pie started throwing weird errors when called without its `data` prop. I imagine we might want it to fail with an invariant?

Reply:
Yeah I'd imagine so. Might as well! Loud and clear is better than a broken graph